ynr/apps/people/tests/test_person_models.py

```diff
@@ -121,3 +123,22 @@ def test_something_goes_wrong_with_delete(self):
                         action_type=ActionType.PERSON_DELETE,
                     ).exists()
                 )
+
+    def test_queued_image(self):
+        user = get_user_model().objects.create()
+        person = PersonFactory()
+        QueuedImage.objects.create(
+            user=user, person=person, image=EXAMPLE_IMAGE_FILENAME
+        )
+        self.assertTrue(person.queued_image)
```

Comment:
A more accurate assert here would be to assign the create `QueudImage` to a variable and then use it with `self.assertEqual`:
```
QueuedImage.objects.create(
     user=user, person=person, image=EXAMPLE_IMAGE_FILENAME
 )
 self.assertEqual(person.queued_image, expected)
 ```